Akka.Serialization.MessagePack/Internal/TypeDictCtr.cs

```diff
@@ -5,7 +5,7 @@ namespace Akka.Serialization.MessagePack.Resolvers;
 /// </summary>
 internal static class TypeDictCtr
 {
-    private static int _typeCtr;
+    private static int _typeCtr = -1;
```

Comment:
`Interlocked.Increment` returns the incremented value,
So we want to have 0 be the first returned.